### PR TITLE
Fixes issue #65, adds support for emotes/anims in dialogue

### PR DESCRIPTION
**Pull request by ryleykimmel**

### CODE REVIEW

**Comment by Major-** [comment]
Don't merge, block comment not yet addressed

**Comment by Major-** [comment]
Also please squash when block nit is address

**Comment by ryleykimmel** [comment]
Unsure of what you want me to do for the block, unable to get it working the way you suggested.

File: data/plugins/dialogue/dialogue.rb [hunk 1]
```diff
@@ -419,23 +419,32 @@ def send_text_dialogue(player, dialogue)
...
 
 # Sends a dialogue displaying the head of an npc.
 def send_npc_dialogue(player, dialogue)
   npc = dialogue.npc
+  emote = dialogue.emote
   name = NpcDefinition.lookup(npc).name.to_s
   name = "" if (name.nil? || name == "null")
 
-  send_generic_dialogue(player, dialogue, name, NPC_DIALOGUE_IDS, ->(id) { SetWidgetNpcModelMessage.new(id + 1, npc)})
+  send_generic_dialogue(player, dialogue, name, NPC_DIALOGUE_IDS, ->(id) {
```
Comment (garyttierney) [inline data/plugins/dialogue/dialogue.rb]:
```ruby
send_generic_dialogue player, dialogue, name, NPC_DIALOGUE_IDS, do |id|
   # ...
end
```
is not working?

Comment (ryleykimmel) [inline data/plugins/dialogue/dialogue.rb]: I kept the parenthesis there, heheheh. My ruby sucks ty.

**Comment by ryleykimmel** [comment]
Squashed and ready for merge

**Pull request merged**